gaurun/worker.go

```diff
@@ -3,53 +3,94 @@ package gaurun
 import (
 	"github.com/RobotsAndPencils/buford/push"
 	"strings"
+	"sync/atomic"
 )
 
+var (
+	PusherCount int64
```

Comment:
IMO, the difference between sharing `PushCount` and providing `PushCount` by each worker is moderate.